### DISCUSSION TITLE
sentry-compose-android missing from BOM

### DISCUSSION THREAD

**Discussion by icsports** [getsentry/sentry-java#2503] [General]
I'm not sure if this is a bug or done by design for some reason, but `sentry-compose-android` (and `sentry-compose-desktop`) appear to be missing from the BOM
Is there any info on whether these should/shouldn't be a part of the BOM?

**Comment by romtsn** [comment]
Nope, most likely we have just overlooked it, sorry for that! Although it should be picked up automatically, we'll investigate why it's not there and provide a fix in the next version.

**Comment by romtsn** [comment]
Opened https://github.com/getsentry/sentry-java/pull/2504 to fix this. Thanks for reporting! Should be fixed with the next released version.
